galsim/chromatic.py

```diff
@@ -1359,3 +1746,74 @@ def withScaledFlux(self, flux_ratio):
             return ChromaticAutoCorrelation( self.obj * math.sqrt(flux_ratio), **self.kwargs )
         else:
             return ChromaticObject(self).withScaledFlux(flux_ratio)
+
+class ChromaticOpticalPSF(ChromaticObject):
+    """A subclass of ChromaticObject meant to represent chromatic optical PSFs.
+
+    Chromaticity plays two roles in optical PSFs. First, it determines the diffraction limit, via
+    the wavelength/diameter factor.  Second, aberrations such as defocus, coma, etc. are typically
+    defined in physical distances, but their impact on the PSF depends on their size in units of
+    wavelength.  Other aspects of the optical PSF are achromatic, e.g., the obscuration and struts.
+
```

Comment:
This last sentence is a bit confusing.  I think the obscuration and struts are chromatic (since they're diffractive effects), but since they're specified in units of the aperture diameter, and the diameter is specified chromatically, these get correctly implemented without any additional wavelength-dependence.